pkg/ui/src/Toast/Toast.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import { fade } from 'svelte/transition';
+
+	import { X } from 'lucide-svelte';
+
+	import { ToastType } from './enums';
+
+	const dispatch = createEventDispatcher();
+
+	export let type: ToastType;
+	export let dismissible = true;
+
+	const colourLookup = {
+		[ToastType.Error]: 'bg-red-50 text-red-900',
+		[ToastType.Success]: 'bg-green-50 text-green-900'
+	};
+</script>
+
+<div
+	class={`flex items-center justify-between max-w-fit px-4 py-2 gap-x-4 shadow-md rounded-lg ${colourLookup[type]}`}
```

Comment:
A small coding style tip: We don't need to use JS string interpolation, can just do Svelte string interpolation:

Instead of:
```jsx
<div
	class={`flex items-center justify-between max-w-fit px-4 py-2 gap-x-4 shadow-md rounded-lg ${colourLookup[type]}`}
  />
```

you can do:
```svelte
<div
	class="flex items-center justify-between max-w-fit px-4 py-2 gap-x-4 shadow-md rounded-lg {colourLookup[type]}"
  />
```

Reply:
_(This only works in .svelte files)_

Reply:
Noted, thank you! Really hated the look of the string interpolation in these cases

Reply:
Rich Harris feels you 😉  (there was a whole discussion on the matter haha)